src/test/java/com/revature/controllers/ReviewControllerTest.java

```diff
@@ -93,7 +99,9 @@ void testGetReviews_Success() throws Exception {
 
 		given(this.rServ.findAll()).willReturn(reviews);
 
-		MockHttpServletRequestBuilder request = get(this.MAPPING_ROOT).accept(MediaType.APPLICATION_JSON);
+		MockHttpServletRequestBuilder request = get(this.MAPPING_ROOT)
+				.accept(MediaType.APPLICATION_JSON)
+				.header(HttpHeaders.AUTHORIZATION, "Bearer token");
```

Comment:
It would be nice to store the fake bearer token as a final String property, just like we did with the controller's mapping root URL